frontend/components/nav/NavInfoButton.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+
+</script>
+
+<template>
+  <UButton class="dark:bg-primary-800 hover:text-white dark:text-white text-primary-500 bg-primary-100/75 max-w-xs transition duration-300 ease-in-out hover:scale-110">Info</UButton>
+</template>
+
+<style scoped>
+
+</style>
```

Comment:
Where is the implementation?
Please read the issue description!!

![Screenshot 2024-03-21 195321](https://github.com/Dino-Kupinic/Schulbuchaktion/assets/87943435/1934aa7d-949d-435b-9e56-3355fd112efe)